users/dierkes/preprocessing/wav2vec.py

```diff
@@ -0,0 +1,69 @@
+import soundfile
+import glob
+import os
+import random
+
+from sisyphus import Job, Task, tk
+
+class FairseqWav2VecManifestCreationJob(Job):
```

Comment:
Is the created manifest directly related to wav2vec or do other models/setups use the same?

Reply:
It is contained in the example folder only for wav2vec but they seem to use it in other examples as well, e.g. data2vec. Should I rename the Job to something more general? Maybe `FairseqRawAudioManifestCreationJob`, `FairseqRawAudioFileCreationJob` or just `FairseqManifestCreationJob`?

Reply:
`FairseqAudioManifestCreationJob`?

Reply:
Ok